@here/harp-vectortile-datasource/lib/VectorTileDataSource.ts

```diff
@@ -146,15 +146,26 @@ export interface VectorTileDataSourceParameters extends DataSourceOptions {
     copyrightProvider?: CopyrightProvider;
 
     /**
-     * Maximum geometry height above groud level this `OmvDataSource` can produce.
+     * Maximum geometry height above ground level this `OmvDataSource` can produce.
      *
-     * @renarks
+     * @remarks
      * Used in first stage of frustum culling before [[Tile.maxGeometryHeight]] data is available.
      *
      * @default [[EarthConstants.MAX_BUILDING_HEIGHT]].
      */
     maxGeometryHeight?: number;
 
+    /**
+     * Minimum geometry height below ground level this `OmvDataSource` can produce. Negative values
+     * describe height below ground.
+     *
+     * @remarks
+     * Used in first stage of frustum culling before [[Tile.minGeometryHeight]] data is available.
+     *
+     * @default `0`.
+     */
+    minGeometryHeight?: number;
```

Comment:
This should not be here but in `DataSourceOptions`. Same for `maxGeometryHeight`

Reply:
Done.